flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/JobMasterPartitionTrackerImplTest.java

```diff
@@ -73,7 +77,7 @@ private static void testReleaseOnConsumptionHandling(ResultPartitionType resultP
         partitionTracker.startTrackingPartition(
                 resourceId,
                 AbstractPartitionTrackerTest.createResultPartitionDeploymentDescriptor(
```

Comment:
Please review and check this change  -- since `JobMasterPartitionTrackerImpl#partitionTable` only tracks internal partitions, I updated field of `hasLocalResources` as true.